### PR TITLE
Add links to OSGeoLive branding

### PR DESCRIPTION
**Pull request by bencaradocdavies**
Matches latest discussion in which project was named "OSGeoLive":
https://lists.osgeo.org/pipermail/live-demo/2017-November/012873.html
http://irclogs.geoapt.com/osgeolive/%23osgeolive.2017-11-06.log

Links will work when https://github.com/OSGeo/OSGeoLive-doc/pull/339 is merged.

### CODE REVIEW

**Comment by bencaradocdavies** [comment]
@jodygarnett who can review this PR?

File: marketing/branding/subbrands/live/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# OSGeo Live
+# OSGeoLive
 
-The OSGeo Live project has decided not to use this subbrand and adopt a distinct brand that more accurately reflects their projects function.
\ No newline at end of file
+The OSGeoLive project has decided not to use this subbrand and adopt a distinct brand that more accurately reflects their projects function.
+
+OSGeoLive branding is maintained in the [OSGeoLive documentation repository](https://github.com/OSGeo/OSGeoLive-doc/tree/master/images/osgeolive-logo).
```
Comment (jodygarnett) [inline marketing/branding/subbrands/live/README.md]: Can we include the image visually in the README?

**Comment by jodygarnett** [comment]
Reviewed, one question.

@bencaradocdavies the marketing committee set this repo up for collaboration with committees and local chapters. If you would like to be added to the team at any point pease ask.

**Comment by bencaradocdavies** [comment]
@jodygarnett I have added the OSGeoLive logo to the README.

**Comment by bencaradocdavies** [comment]
Thanks @jodygarnett.

**Pull request merged**